src/deluge/gui/ui/keyboard/layout/column_controls.cpp

```diff
@@ -53,7 +54,7 @@ void ColumnControlsKeyboard::enableNote(uint8_t note, uint8_t velocity) {
 
 	ColumnControlState& state = getState().columnControl;
 
-	for (int i = 0; i < MAX_CHORD_NOTES; i++) {
+	for (int i = 1; i < MAX_CHORD_NOTES; i++) {
```

Comment:
why isn't this starting from 0, won't that make it never play the root note?

Reply:
because this class was sending the root note 2 lines before that for loop
	currentNotesState.enableNote(note, velocity);
and I changed the definition of chords to contain the root note (for the arpeggiator to use), because they initially started in the second note.